transport-native-epoll/pom.xml

```diff
@@ -179,7 +179,7 @@
                       <addDefaultImplementationEntries>true</addDefaultImplementationEntries>
                     </manifest>
                     <manifestEntries>
-                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_epoll.so; osname=linux; processor=x86_64,*</Bundle-NativeCode>
+                      <Bundle-NativeCode>META-INF/native/libnetty_transport_native_epoll_${os.detected.arch}.so; osname=linux; processor=${os.detected.arch},*</Bundle-NativeCode>
```

Comment:
:ok_hand:

Reply:
This will break folks that are shading ... I don't have a problem with this but just good to note.

Reply:
@Scottmitch imho how these files are named is an implementation detail so it should be ok ?

Reply:
When shading you have to rename them, that means having some level of knowledge how they are named.

The recent change from `-` to `_` broke existing configuration for shading as well. Breakages like that should be "okay," but avoiding useless breakages would be good. This breakage seems unavoidable.

Reply:
@ejona86 agree, we should only do if there is a good reason. I think this is one.

Reply:
@normanmaurer worth while a mention in the release notes then :+1: